src/index.html

```diff
@@ -5,10 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img class="header__logo"
+          src="./images/logo.png"
+          alt="moyo-logo"
+        >
```

Comment:
if you are placing attributes on separate lines, it would be better to do this for all attributes. It is more clear for readability
```suggestion
        <img
          class="header__logo"
          src="./images/logo.png"
          alt="moyo-logo"
        >
```

Reply:
Got it! Ty!